deploy/build.yml

```
@@ -21,6 +21,12 @@
       ignore_errors: yes
       tags: 
         - build
+    - name: Build axon
+      shell: "cd {{ deploy_path }} && cargo build --release --features=jemalloc"
```

```suggestion
      shell: "cd {{ deploy_path }} && cargo build --profile release-jemalloc --features=jemalloc"
```

This command will be effective after https://github.com/nervosnetwork/axon/pull/291 is merged.